### ISSUE TITLE
Path with a space is not supported in windows 10

### ISSUE THREAD

**Issue by Anirban20001962** [redhat-developer/rsp-server-community#166]
Error: Could not find or load main class MUKHERJEE\.rsp\redhat-community-server-connector\runtimes\installations\tomcat-7.0.99_2\apache-tomcat-7.0.99.conf.logging.properties
Caused by: java.lang.ClassNotFoundException: MUKHERJEE\.rsp\redhat-community-server-connector\runtimes\installations\tomcat-7.0.99_2\apache-tomcat-7.0.99.conf.logging.properties

I have checked, and it has the required files installed in the path "C:\Users\ANIRBAN MUKHERJEE\.rsp\redhat-community-server-connector\servers".

Please make changes so that I can use this magnificent extension.

**Comment by robstryker** [comment]
Hi @Anirban20001962 

I'm unable to replicate this issue. The relevant code sets out a template for the start command, and it looks like this:
```
                                                        "vmArgs": "-Djava.util.logging.config.file='${server.base.dir}/conf/logging.properties' -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base='${server.base.dir}' -Dcatalina.home='${server.home.dir}' -Djava.io.tmpdir='${server.base.dir}/temp'",
```
You can see here that all instances of a path are prefixed with an apostrophe to be space-safe. 

But I notice that your command line seems to be looking for a file named `apache-tomcat-7.0.99.conf.logging.properties` which is not part of our default launch script. Did you, by chance, right click on the server, select `Edit Server`, and then change the `args.vm.override.string` property manually? If you did, can you ensure you used apostraphes inside? 

For reference, this is what my property looks like:

`"args.vm.override.string": "-Djava.util.logging.config.file='/conf/logging.properties' -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base='' -Dcatalina.home='/home/rob/apps/apache/tomcat/with space/apache-tomcat-10.1.4' -Djava.io.tmpdir='/temp'",`

**Comment by Anirban20001962** [comment]
The problem is that there is space in the username. So it cannot find the file although it is there.

**Comment by robstryker** [comment]
I understand that a space would be likely to cause this issue, but a space in the user name is no different than a space in any folder and I've tested with spaces in arbitrary folders. 

It's clear the issue is with the `-Djava.util.logging.config.file` property. What I can see is that back in `Date:   Thu Nov 3 15:44:40 2022 -0400` I made sure to wrap that command line argument / system property with quotes to ensure the path with the space is treated as a single path and not misinterpreted by the command line. 

You can see that commit here:  https://github.com/redhat-developer/rsp-server-community/commit/b0e0d07c81923fae8e39585dacea81e01bc9de60

Can you verify what version of the community-server-connector you are using? Can you also verify if the server was created / downloaded with a recent installation and not an old one? 

I've done another release today. Maybe you can try with the newest.

**Comment by robstryker** [comment]
Hi:

This was likely fixed by * f755455 - (HEAD -> master, origin/master) Fixes issue #174 - single qutoes don't work on windows (3 weeks ago) <Rob Stryker>
